app/code/Magento/Catalog/Test/Unit/Helper/Product/ProductListTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Test\Unit\Helper\Product;
+
+use Magento\Catalog\Helper\Product\ProductList;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Magento\Catalog\Model\Category\Toolbar\Config as ToolbarConfig;
+
+class ProductListTest extends TestCase
+{
+    /**
+     * @var ProductList
+     */
+    private $object;
+
+    /**
+     * @var ScopeConfigInterface|MockObject
+     */
+    private $scopeConfig;
```

Comment:
Currently, we are advocating suffixing mocks with the `Mock` keyword. So, it's easier to differentiate mocks and real objects through the test. It's not a requirement, but if you could follow this practice, I would appreciate it.

```php
private $scopeConfigMock;
private $toolbarConfigMock;
```

Reply:
@rogyar no problem at all. Can you please refer to documentation to read more about the `Mock` suffix?

Thank you in advance.

Reply:
Unfortunately, there's no documentation that explicitly describes the best practices for creating unit tests in DevDocs so far. But we are working on the topic on that matter.